orchagent/portsorch.cpp

```diff
@@ -1944,8 +1944,37 @@ void PortsOrch::initPortCapAutoNeg(Port &port)
     }
     else
     {
-        port.m_cap_an = 0;
-        SWSS_LOG_NOTICE("Unable to get %s AN capability", port.m_alias.c_str());
+        // To avoid breakage on the existing platforms, AN should be 1 by default
+        port.m_cap_an = 1;
+        SWSS_LOG_WARN("Unable to get %s AN capability, assumming it's supported",
```

Comment:
I prefer you just add "Unable to get AN support capability" in the log. Please remove "assuming it's supported"

Reply:
Done